site/docs/v1/tech/apis/system.adoc

```diff
@@ -63,7 +63,9 @@ include::docs/v1/tech/apis/_system-configuration-response-body.adoc[]
 
 == Update the System Configuration
 
-This API is used to update System Configuration.
+:capitalized_object_name: System Configuration
+include::docs/v1/tech/apis/_generic-update-explanation-fragment.adoc[]
+:capitalized_object_name!:
```

Comment:
This API will not have a unique Id on the URL.

Reply:
Thanks, handled.